tests/integration/components/to-elsewhere-test.js

```diff
@@ -55,4 +57,13 @@ module('Integration | Component | to elsewhere', function(hooks) {
     assert.dom(this.element.querySelector('.my-target')).hasText('Hello World from Foo');
   });
 
+  test('it accepts an outsideParams object for inline form', async function(assert) {
+    await render(hbs`<div class="source">{{to-elsewhere named="my-target" send=(component "x-baz") outsideParams=(hash greeting="Hello World")}}</div><div class="my-target">{{from-elsewhere name="my-target"}}</div>`);
+    assert.dom(this.element.querySelector('.my-target')).hasText('Hello World from Baz');
```

Comment:
And so this test would go away. The inline form doesn't have access to `outsideParams`.